clients/admin-ui/src/features/privacy-experience/Preview.tsx

```diff
@@ -60,6 +60,27 @@ const NoPreviewNotice = () => (
   </Flex>
 );
 
+const TCFExperienceNotice = () => (
+  <Flex
+    bgColor="white"
+    borderRadius="md"
+    p={6}
+    boxShadow="md"
+    direction="column"
+    align="center"
+    gap="2"
+    maxW="512px"
+  >
+    <Text fontSize="lg" fontWeight="500" align="center">
+      TCF preview not available
+    </Text>
+    <Text color="gray.500" align="center">
+      There is no preview available for TCF. You can edit the available settings
+      and languages to the left.
+    </Text>
```

Comment:
can we extract / share a component with https://github.com/ethyca/fides/pull/4650/files#diff-7344022362ed6489650e4804f41c1c0cb81687fe5396c525ce66a404a3a12cccR53-R60 for preview not available?